bentoml/_internal/server/__init__.py

```diff
@@ -1,21 +1,33 @@
 import os
 import sys
 import json
+import atexit
 import shutil
 import typing as t
 import logging
 import tempfile
 import contextlib
+from datetime import datetime
+from datetime import timezone
 
 import psutil
 from simple_di import inject
 from simple_di import Provide
 
 from bentoml import load
-from bentoml._internal.utils import reserve_free_port
-from bentoml._internal.utils.uri import path_to_uri
-from bentoml._internal.utils.circus import create_standalone_arbiter
 
+from ..utils import reserve_free_port
+from ..utils.uri import path_to_uri
+from ..utils.circus import create_standalone_arbiter
+from ..utils.analytics import track
+from ..utils.analytics import get_serve_info
+from ..utils.analytics import scheduled_track
+from ..utils.analytics import BentoServeProductionOnStartupEvent
```

Comment:
Suggest renaming events:
```
ServeStartEvent
ServeUpdateEvent
ServeEndEvent
ServeDevStartEvent
ServeDevUpdateEvent
ServeDevEndEvent
```

Reply:
I think `ServeProductionStartEvent` would be more explicit?